### ISSUE TITLE
Login Page: Fix axe violations

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#3417]
**Describe the issue. What is the expected and unexpected behavior?**
- Links must have discernible text (issue in example links)

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

Run axe extension

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Enhancement- doesn't seem as critical due to the fact that it's in an example and unlikely to be needed by a user

**What is your product and what release version are you targeting?**

**Comment by jessiehuff** [comment]
Seems to be resolved, likely closed by #3593
